src/UI/state/home/actions/getHomepageStats.ts

```diff
@@ -0,0 +1,17 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { fetchGraphQlData } from '../../../api/api';
+import { getHomepageAnalytics } from '../../../api/queries';
+import { updateStats } from '../homeSlice';
+
+const date = new Date();
+
+export const getHomepageStats = createAsyncThunk(
+  'home/getHomepageStats',
+  async (_, thunkAPI) => {
+    const response = await fetchGraphQlData(
+      getHomepageAnalytics(0, date.getTime(), 'hour', 'Europe%2FLondon')
+    );
+    const data = response.data.getHomepageAnalytics;
+    thunkAPI.dispatch(updateStats(data));
```

Comment:
What happens here if `getHomepageAnalytics` fails? The analytics server might be down for some reason, which would probably cause this call to fail. Does the UI then fall over as well? We probably just want it to not display the stats if this fails.